packages/repack/tsconfig.json

```diff
@@ -69,6 +69,6 @@
     "skipLibCheck": true,                           /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
   },  
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "js/NativeScriptManager.ts"],
```

Comment:
Why `NativeScriptManager` is added as a separate file to the config? We should include all ts files in the config

Reply:
Will fix. Thanks

Reply:
Fixed.